src/components/projecteditor/ProjectEditor.tsx

```diff
@@ -118,7 +119,7 @@ export class ProjectEditor extends React.Component<IProps, IState> {
                                         <Panes dragging={sidePanelDragging} />
 
                                         { this.isPanelOpen(Panels.Transactions) &&
-                                        <Panel icon={ <IconTransactions /> } name='Transactions History' onClose={() => closePanel(Panels.Transactions)} dragging={sidePanelDragging}>
+                                        <Panel name='Transactions History' onClose={() => closePanel(Panels.Transactions)} dragging={sidePanelDragging}>
```

Comment:
Question here - is it as intended to remove icons from panels?

Reply:
Yeap, it is intended.

Reply:
Added it to the PR description